src/translations/en_NO.json

```diff
@@ -66,7 +66,7 @@
   "CHECKOUT_EMAIL_PLACEHOLDER": "your.email@here.com",
   "CHECKOUT_SUMMARY_TITLE": "Your details",
   "CHECKOUT_SIGN_BUTTON_TEXT": "Sign with BankID",
-  "CHECKOUT_SIGN_DISCLAIMER": "By clicking “Sign with BankID” I confirm that I have read and understood [the pre-sale information]({PREBUY_LINK}), [terms]({TERMS_LINK}) and that my personal data is handled [according to GDPR](https:\/\/www.hedvig.com\/privacy).",
+  "CHECKOUT_SIGN_DISCLAIMER": "",
```

Comment:
Shouldn't we still have this disclaimer in Norway? But replace "Sign with BankID" with just "Sign"

Reply:
Aha, no there is another one specifically for Norway. It's called `CHECKOUT_SIGN_DISCLAIMER_NO`

Reply:
Aha, cool! 🙌